src/uinput.rs

```diff
@@ -7,10 +7,7 @@ use crate::constants::{EventType, UInputEventType};
 use crate::ff::FFEffectData;
 use crate::inputid::{BusType, InputId};
 use crate::raw_stream::vec_spare_capacity_mut;
-use crate::{
-    sys, AttributeSetRef, Error, FFEffectType, InputEvent, InputEventKind, Key, RelativeAxisType,
-    SwitchType, UinputAbsSetup,
-};
+use crate::{sys, AttributeSetRef, Error, FFEffectType, InputEvent, InputEventKind, Key, RelativeAxisType, SwitchType, UinputAbsSetup, PropType};
```

Comment:
This got longer, and the includes are out of alpha order. Did you run `cargo fmt` yet?

Reply:
thanks for noticing, I had intellij configured to run optimize imports on commit I think.